### PR TITLE
[v2.0.3] : Fixed issue #2929, #3149

### PR DESCRIPTION
**Pull request by monica-BTC**
- Fixed issue #2929, by standardizing use of title case and lower case across screens, and updated text accordingly.
- Fixed issue #3149, by removing attribute `reasonForConsent` in ReviewBean.java

### CODE REVIEW

File: study-datastore/src/main/java/com/hphc/mystudies/bean/ReviewBean.java [hunk 1]
```diff
@@ -24,18 +24,8 @@
 
 public class ReviewBean {
 
-  private String reasonForConsent = "";
```
Comment (moschetti) [inline study-datastore/src/main/java/com/hphc/mystudies/bean/ReviewBean.java:27]: Please update file copyright to add Google LLC & `2020-2021`

File: study-datastore/src/main/java/com/hphc/mystudies/dao/StudyMetaDataDao.java [hunk 2]
```diff
@@ -740,11 +740,6 @@ public EligibilityConsentResponse eligibilityConsentMetadata(String studyId) thr
                           .replaceAll("em>", "i>")
                           .replaceAll("<a", "<a style='text-decoration:underline;color:blue;'"));
             }
-
```
Comment (moschetti) [inline study-datastore/src/main/java/com/hphc/mystudies/dao/StudyMetaDataDao.java:743]: Please update file copyright to add Google LLC & `2020-2021`

Comment (monica-BTC) [reply study-datastore/src/main/java/com/hphc/mystudies/dao/StudyMetaDataDao.java:743]: Updated copyright.

File: study-datastore/src/main/java/com/hphc/mystudies/bean/ReviewBean.java (hunk 1 above)
Comment (monica-BTC) [reply study-datastore/src/main/java/com/hphc/mystudies/bean/ReviewBean.java:27]: Updated copyright.

**Review by moschetti — DISMISSED** [review]

**Comment by monica-BTC** [comment]
@moschetti, resolved conflicts please approve the PR.

File: study-builder/fdahpStudyDesigner/src/main/java/com/fdahpstudydesigner/bo/ConsentBo.java [hunk 3]
```diff
@@ -48,6 +48,7 @@
 
   private static final long serialVersionUID = 5564057544960167010L;
 
+  @Deprecated
```
Comment (moschetti) [inline study-builder/fdahpStudyDesigner/src/main/java/com/fdahpstudydesigner/bo/ConsentBo.java:51]: add google 2020-2021 to copyright like other files

Comment (monica-BTC) [reply study-builder/fdahpStudyDesigner/src/main/java/com/fdahpstudydesigner/bo/ConsentBo.java:52]: Fixed review comment.

**Review by moschetti — DISMISSED** [review]

**Comment by monica-BTC** [comment]
@moschetti , resolved conflicts please reapprove the PR.

**Review by moschetti — APPROVED** [review]

**Pull request merged**